app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str):
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str):
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self):
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.skills = ["Python", "SQL", "Django"]
+
+    def create_powerful_api(self):
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.skills = ["Java", "Android studio"]
```

Comment:
Add the list of the required skills to the attribute. In that case, you shouldn't create a constructor for `FullStackDeveloper` class
```suggestion
        self.skills += ["Java", "Android studio"]
```